cmd/ebrelayer/main.go

```diff
@@ -287,6 +299,7 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 		web3Provider,
 		contractAddress,
 		sugaredLogger,
+		sifnodeGrpc,
```

Comment:
It's so good when you can put a comma at the end of a list before before the closing bracket. I'm looking at you JSON.